scss-c2s/c2s-results.scss

```diff
@@ -0,0 +1,379 @@
+@mixin media($size: false, $max-width: false, $min-width: false, $ignore-for-ie: false) {
+  @if $size == desktop {
+      @media (min-width: 925px) {
+        @content;
+      }
+    } @else if $size == tablet {
+      @media (min-width: 641px) {
+        @content;
+      }
+    } @else if $size == mobile {
+      @media (max-width: 640px) {
+        @content;
+      }
+    } @else if $max-width != false {
+      @media (max-width: $max-width) {
+        @content;
+      }
+    } @else if $min-width != false {
+      @media (min-width: $min-width) {
+        @content;
+      }
+    } @else {
+      @media (min-width: $size) {
+        @content;
+      }
+    }
+}
+
+$nhs-blue: #0D60B5;
```

Comment:
Are these colours the new versions of the colours in `scss-live/environment/settings/_colours.scss`? Is the intention to update the front-end library at some point?

Reply:
I'm not adding any of this work into the FEL until it's been full agreed and signed off. It's very much WIP ATM.